net-libs/libnice/patches/libnice_x86-0.1.13.patchset

```diff
@@ -0,0 +1,93 @@
+From d5f26c32ad9d2c2a3f345504f58244f9bc110330 Mon Sep 17 00:00:00 2001
+From: Steven Kou <kouyongkang@gmail.com>
+Date: Sat, 26 Dec 2015 23:18:02 +0000
+Subject: Search for socket under network instead of inet
+
+
+diff --git a/configure.ac b/configure.ac
+index 223b883..f496ec0 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -167,7 +167,7 @@ AS_IF([test "x$enable_compile_warnings" = "xerror"],[
+ # Fixes for Solaris
+ #
+ AC_SEARCH_LIBS([inet_pton],[nsl])
+-AC_SEARCH_LIBS([socket],[socket inet])
++AC_SEARCH_LIBS([socket],[socket network])
```

Comment:
According to documentation (https://www.gnu.org/software/autoconf/manual/autoconf-2.66/html_node/Libraries.html) you can add `network` to the list instead of replacing `inet`.